flux/stdlib/universe/merge_filter_test.flux

```diff
@@ -0,0 +1,50 @@
+package universe_test
```

Comment:
This test was in the Flux repo, but it was recently commented out (I assume because IDPE hasn't implemented the pushdown yet). Transferred it here so we're protected against regressions on this rewrite rule.